app/models/product.rb

```diff
@@ -215,13 +215,15 @@ def stage
     end
 
     def write_manifest
+      manifest = Manifest.load( router.path.join('manifest.yml') )
+      if manifest.is_a?(InvalidManifest)
+        manifest = Manifest.new({name: ""})
+      end
+      manifest.name = title unless title.blank?
+      manifest.description = description unless description.blank?
       File.open(router.path.join('manifest.yml'), 'w') do |f|
-        f.write({
-          'name' => title,
-          'description' => description
-        }.to_yaml)
-      end if (!title.blank? || !description.blank?) || !router.path.join('manifest.yml').exist?
-      true
+        f.write(manifest.to_yaml)
+      end if router.path.join('manifest.yml').exist?
```

Comment:
Don't force generating a manifest on `git clone`. The manifest is corrupt when clicking "Build New App" but we just need to fix the manifest in the starter app.